content/src/cloud/troubleshooting-cloud-backup/backup-is-not-encrypted.md

```diff
@@ -6,7 +6,17 @@ zendesk:
   labels: cloud
 ---
 
-**Backup is not encrypted** - If you are using an Automation, Blueprint, or Add-on to backup your system on a scheduled basis, and encryption is not set, those backups will not be uploaded. Set a password on those backups if you want them uploaded to Home Assistant Cloud.
+## Symptom
+
+In Home Assistant, you're seeing a notification **Backup is not encrypted**.
+
+## Cause
+
+If you are using an automation, blueprint, or add-on to create backups of your system on a scheduled basis, and encryption is not set, those backups will not be uploaded to Home Assistant Cloud.
+
+## Remedy
+
+Set a password on those backups if you want them uploaded to Home Assistant Cloud.
```

Comment:
They can also exclude cloud as target if they don't want to use encryption for this specific one.